docs/recipe-spec/recipe-spec.md

```diff
@@ -34,6 +34,35 @@ description: string, required
 # Example: https://github.com/newrelic/infrastructure-agent
 repository: string, required
 
+# Indicates stability level of recipe. Useful for gradually enabling the
+# availablity of a recipe to users as it is developed and tested. Can be
+# thought of as an indicator for how stable/ready a recipe is for general 
+# consumption and can be used to quickly disable problematic recipes.
+# 
+# Usage: # newrelic install --stability=stable
+# 
+# Levels:
+#   disabled - No availability (recipe will not return from NerdGraph)
+#   experimental - Non-backward compatible changes or removal may occur in any future release. Use of the feature is not recommended in production environments.
+#   stable - General availability - ongoing support and compatability is a high priority
+stability: enum, optional # One of [ disabled, experimental, stable ]
```

Comment:
I'm not onboard with this attribute, I don't want recipe out with less clarity on their reliability, if a customer tries out something that's disabled or experimental that is still something we'll have to support.
We don't need this for making Infra optional.

Reply:
@Julien4218 This doesn't have to do with the Infra MMF. It's for [Virutoso-616](https://newrelic.atlassian.net/browse/VIRTUOSO-616), and it's tangentially related to the Quickstarts work so that we have a way to start adding that into experimental recipes.

We've agreed previously that we should have some mechanism for gating recipes. I don't think this does anything w.r.t. clarity around reliability, it's just a way to limit what recipes are used in a given context via a CLI flag. It's mostly a way for us to experiment without making a recipe generally available.